drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenterTest.java

```diff
@@ -1360,9 +1374,12 @@ public void onDeleteSelectedCellsWithSelection() {
 
     @Test
     public void onDeleteSelectedCellsWithSelectionWithBooleanColumn() {
-        final AttributeCol52 column = new AttributeCol52() {{
-            setAttribute(RuleAttributeWidget.ENABLED_ATTR);
-        }};
+        final AttributeCol52 column = new AttributeCol52() {
```

Comment:
Is this Eclipse's formatting of static initialiser blocks (in anonymous sub-classes!)